test/integration/share/README.md

```diff
@@ -0,0 +1,20 @@
+# InSpec Profile: `share`
+
+This shows the implementation of the `share` InSpec [profile](https://github.com/inspec/inspec/blob/master/docs/profiles.md).
+
+It's goal is to share the libraries between all profiles.
+
+## Libraries
+
+### `system`
+
+The `system` library provides easy access to system dependents informations:
+
+- `system.hostname`: return the result of `hostname -s` or `hostnamectl --static` based on the availability of each commans
+- `system.platform`: take `inspec.platform` and mangle things a bit to be useful
+  - `system.platform[:family]` provides a family name for Arch
+  - `system.platform[:name]` modify `amazon` to `amazonlinux`
+  - `system.platform[:release]` tweak for Arch and Amazon Linux:
+    - `Arch` is always `base-later`
+    - `Amazon Linux` release `2018` became `1`
+  - `system.platform[:finger]` is just the concatenation of the name and the first release number (except for Ubuntu which gives `20.04` for example)
```

Comment:
```suggestion
  - `system.platform[:finger]` is just the concatenation of the name and the first release number (except for Ubuntu which gives `ubuntu-20` for example)
```

Reply:
Actually, this isn't only `ubuntu-20` is it?  It will resolve as `ubuntu-20.04`, right?  If that's the case, then perhaps this rewording is better:

```suggestion
  - `system.platform[:finger]` is the concatenation of the name and the major release number (except for Ubuntu, which gives `ubuntu-20.04` for example)
```